process_report/process_report.py

```diff
@@ -53,8 +60,9 @@ def main():
 
     projects = list(set(projects + timed_projects_list))
 
-    remove_non_billables(merged_dataframe, pi, projects, args.output_file)
+    billable_projects = remove_non_billables(merged_dataframe, pi, projects, args.output_file)
```

Comment:
can we still print the whole billable csv?

Reply:
@naved001 I only modified `remove_non_billable` to return the dataframe with billable projects, so the function should still write out the billable csv

Reply:
oh, right. I missed it, thanks!